templates/ethstore.html

```diff
@@ -0,0 +1,524 @@
+        <div class="content">
+          <div class="table-responsive">
+            <table class="table">
+              <thead>
+                <tr>
+                  <th>Variable</th>
+                  <th>Description</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>Start epoch (SE):</td>
+                  <td>First epoch after 1200 UTC every day</td>
+                </tr>
+                <tr>
+                  <td>End epoch (EE):</td>
+                  <td>SE + 225</td>
+                </tr>
+                <tr>
+                  <td>Active validators (VS):</td>
+                  <td>Validators that have been active from start SE to end EE [SE,EE)</td>
+                </tr>
+                <tr>
+                  <td>Sum rewards (RS):</td>
+                  <td>Sum of all rewards (consensus, tx) in VS that have been generated in epochs [SE,EE)</td>
+                </tr>
+                <tr>
+                  <td>Sum effective balances (EB):</td>
+                  <td>Sum of all effective balances at epoch SE of all validators in VS</td>
+                </tr>
+                <tr>
+                  <td>ETH.STORE:</td>
+                  <td>(RS/EB)*365</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+
+          <br />
+          <p>
+            Calculations for <strong><span aria-ethereum-date="{{ .Data.YesterdayTs }}" aria-ethereum-date-format="dd MMMM yyyy"></span> 12:00:23 UTC - <span aria-ethereum-date="{{ addI64 .Data.YesterdayTs 86400 }}" aria-ethereum-date-format="dd MMMM yyyy"></span> 12:00:22 UTC:</strong>
+          </p>
+
+          <div class="table-responsive">
+            <table class="table">
+              <thead>
+                <tr>
+                  <th>Variable</th>
+                  <th>Description</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>Start epoch (SE):</td>
+                  <td>Epoch {{ .Data.StartEpoch }}</td>
+                </tr>
+                <tr>
+                  <td>End epoch (EE):</td>
+                  <td>Epoch {{ addUI64 .Data.StartEpoch 225 }}</td>
+                </tr>
+                <tr>
+                  <td>Active validators (VS):</td>
+                  <td>in timeframe [{{ .Data.StartEpoch }},{{ addUI64 .Data.StartEpoch 225 }})</td>
+                </tr>
+                <tr>
+                  <td>Sum rewards (RS):</td>
+                  <td>{{ .Data.YesterdayRewards }} Ether</td>
+                </tr>
+                <tr>
+                  <td>Sum effective balances (EB):</td>
+                  <td>{{ .Data.YesterdayEffectiveBalance }} Ether</td>
+                </tr>
+                <tr>
+                  <td>ETH.STORE:</td>
+                  <td>{{ .Data.ProjectedAPR }} % = ({{ .Data.YesterdayRewards }} / ({{ .Data.YesterdayEffectiveBalance }})) * 365</td>
```

Comment:
This page uses the "e+X" notation while https://staking.ethermine.org/statistics uses "^10". Should this be harmonized?

Reply:
Fixed in new commit

Reply:
You have missed one: 
![image](https://user-images.githubusercontent.com/114066135/211492371-db951f54-2dcf-4162-b50f-fbb58c1af2f1.png)
(see the second number)

Also: Does the change with `formatThousandsFancy` work as intended? The numbers seem way too high.

Here is the data for the 4th of January on Prater with your changes:
![image](https://user-images.githubusercontent.com/114066135/211492596-8ec539e4-e266-4a08-8801-ddc10cfb9bc7.png)

And here is the data for the 8th of January on Mainnet:
![image](https://user-images.githubusercontent.com/114066135/211492678-e13cb440-4dda-4433-b73b-9f8737a8cbdb.png)

Compare, for example, the "Sum rewards (RS)".

Reply:
Oversight, sorry! Forgot to convert from Wei to ETH, should be fixed in new commit